### ISSUE TITLE
Re-arrange inner tool bar buttons

### ISSUE THREAD

**Issue by bblackmoor** [tablacus/TablacusExplorer#646]
I have added the specific addons for inner back, forward, refresh, and so on.

![image](https://user-images.githubusercontent.com/33701300/211218381-fd692cb3-44e7-44b3-974b-500273fcb581.png)

How may I re-arrange these icons, please?

![image](https://user-images.githubusercontent.com/33701300/211218397-81cb2750-291a-4a40-96a7-8ee823dc633a.png)

**Comment by bblackmoor** [comment]
I was able to re-arrange the icons by uninstalling the addons, and then re-installing them in a specific order.

![image](https://user-images.githubusercontent.com/33701300/211218833-641d0f51-25b8-4e28-804c-f0c4dd301c32.png)

**Comment by duarteframos** [comment]
There is no need to uninstall and reinstall, you can reorder them preferences.
Most addon's positions are controlled from installed addons list order. Some have a _Postion_ tab in preferences for more fine grained adjustment.

**Comment by bblackmoor** [comment]
I tried disabling and re-enabling, but that did not change the order. I looked into the preferences, and did not see any way to influence the order. I looked at the script.js files, but I did not see anything which I wanted to change. So I tried uninstalling the addons, and installing them in the order I wanted them to appear, and that worked.

Is there another place I should have looked for that configuration?

**Comment by duarteframos** [comment]
Go to _Tools > Addons._

Click on the addon title and drag up or down to reorder, or pick the radio button to the left of any addon you'd like to reoder, and use the up and down arrow buttons at the bottom to reorder.

**Comment by bblackmoor** [comment]
OMG. I had no idea that those were draggable. Thank you!

I did not notice the arrow buttons at the bottom. The user interface might benefit from some small adjustments. If I wanted to make some UI code changes and submit them for merging, how would I do that?

**Comment by duarteframos** [comment]
Project owner is @tablacus Best way is to make changes is to fork it, then make pull requests.
Gaku has been pretty open and welcoming about it, though es probably busy with personal life lately and asn't been very active for the past few months.

I'm not a coder but Ive made a few pull requests myself, to work on improving various things over time.
I made a few icon packs, improved addon descriptions, added wiki pages and documentation among other things.

I know there are some [plans to improve the UI in the future](https://github.com/tablacus/TablacusExplorerIconPacks/issues/10#issuecomment-1044530244), [following some mockups I did a while ago](https://github.com/tablacus/TablacusExplorer/issues/329).
